### ISSUE TITLE
is not working in iOS 14

### ISSUE THREAD

**Issue by MehmetDenizYalcin** [efremidze/VisualEffectView#25]
it is working id you add this

func _setValue<T>(_ value: T, forKey key: Key) {
        blurEffect.setValue(value, forKeyPath: key.rawValue)
        if #available(iOS 14, *) { } else {
            self.effect = blurEffect
        }
    }

**Comment by shkipan** [comment]
Could you be more specific? I have the same issue, recompiling project in IOS14. The blur on view is missing

**Comment by bachvanthe1994** [comment]
I have the same issue, 
But in my app, the blur has shown a black-white view. 
You can see image above
![image](https://user-images.githubusercontent.com/11937975/93727568-dfdb5f80-fbe5-11ea-9c4b-ed1dbdaf51ac.png)

**Comment by MehmetDenizYalcin** [comment]
@bachvanthe1994 yes if you add latest changes and change below code like this it will work 

func setValue( value: T, forKey key: Key) {
    blurEffect.setValue(value, forKeyPath: key.rawValue)
     if #available(iOS 14, *) { } else {
    self.effect = blurEffect
 }
}

self.effect = blurEffect remove _UIVisualEffectSubview and you see black and white in iOS 14

**Comment by bachvanthe1994** [comment]
> @bachvanthe1994 yes if you add latest changes and change below code like this it will work
> 
> func setValue( value: T, forKey key: Key) {
> blurEffect.setValue(value, forKeyPath: key.rawValue)
> if #available(iOS 14, *) { } else {
> self.effect = blurEffect
> }
> }
> 
> @shkipan self.effect = blurEffect remove _UIVisualEffectSubview and you see black and white in iOS 14

Thank bro, that worked. But I hope the owner of the respository will commit a new release. I don't want to change the source in pod folder because it is not stability

**Comment by MehmetDenizYalcin** [comment]
> Hey guys, was wondering if any of you could potentially help me please as ive been stuck on this issue for a while.
> 
> I cant seem the blur effect to work on iOS14 devices, it works fine on iOS13 though.
> 
> ```
>     let blurEffect = (NSClassFromString("_UICustomBlurEffect") as! UIBlurEffect.Type).init()
>     blurEffect.setValue(8, forKeyPath: "blurRadius")
>     blurView = UIVisualEffectView(effect: blurEffect)
>     blurView.frame = UIScreen.main.bounds
>     blurView.alpha = 1
>     view.insertSubview(blurView, aboveSubview: imvBackground)
> ```
> 
> Is anyone please able to help me fix this? Thanks.

Check my comment, it is the first one.

**Comment by nadimalam** [comment]
> > Hey guys, was wondering if any of you could potentially help me please as ive been stuck on this issue for a while.
> > I cant seem the blur effect to work on iOS14 devices, it works fine on iOS13 though.
> > ```
> >     let blurEffect = (NSClassFromString("_UICustomBlurEffect") as! UIBlurEffect.Type).init()
> >     blurEffect.setValue(8, forKeyPath: "blurRadius")
> >     blurView = UIVisualEffectView(effect: blurEffect)
> >     blurView.frame = UIScreen.main.bounds
> >     blurView.alpha = 1
> >     view.insertSubview(blurView, aboveSubview: imvBackground)
> > ```
> > 
> > 
> > Is anyone please able to help me fix this? Thanks.
> 
> Check my comment, it is the first one.

Oh right, thank you, but please can you tell me where to add this setValue function?
